app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  protected
+
+  def configure_permitted_parameters
+    # Permit the username
+    # sign up parameters.
```

Comment:
You should probably delete this comment. The reason being is that this may change over time, so you would need to maintain the comments themselves as well, which we have no good way of keeping track of.